frontend/public/components/default-resource.jsx

```diff
@@ -137,7 +139,12 @@ export const DefaultPage = (props) => (
 DefaultPage.displayName = 'DefaultPage';
 
 export const DefaultDetailsPage = (props) => {
-  const pages = [navFactory.details(DetailsForKind(props.kind)), navFactory.editYaml()];
+  const pages = [
+    navFactory.details(DetailsForKind(props.kind)),
+    navFactory.editYaml(
+      kindForReference(props.kind) === 'Task' ? viewYamlComponent : editYamlComponent,
```

Comment:
Can we use `TaskModel.kind` instead of `'Task'` ?

Reply:
This change is not valid anymore. Added a new page for task run overview.